releases/R2023b/README.md

```diff
@@ -43,10 +42,15 @@ To deploy MATLAB Production Server onto an existing VPC, specify these additiona
 
 | Parameter Name | Value |
 |--------------- | ----- |
-| Existing VPC ID | ID of your existing VPC. |
+||**Network License Manager**|
+| **Deploy License Server** | Specify whether you want to deploy the Network License Manager for MATLAB. This parameter is available only if you use the deployment template for an existing VPC. <p>You can deploy a license server only if your solution uses public IP addresses. If your solution uses private IP addresses, you must separately deploy a license server in a public subnet.</p><p>If you are using an existing VPC, see the [Use Existing License Server in Existing VPC](#use-existing-license-server-in-existing-vpc) section.</p> |
```

Comment:
This statement is no longer true. Starting in 23b, you can deploy license server in a private subnet also. 
We can remove these sentences: "You can deploy a license server only if your solution uses public IP addresses. If your solution uses private IP addresses, you must separately deploy a license server in a public subnet"